.changes/next-release/Bug Fix-880d1db0-1a68-4fbb-96c8-af1a988238bb.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "CodeWhisperer: fix error exceed max supplemental context length"
```

How does the customer perceive the issue? Will this description mean anything to a customer? 

If the impact is something like "CodeWhisperer sometimes fails on big files", then that is what the changelog item should say.
```suggestion
	"description": "CodeWhisperer sometimes fails on big files"
```

---

sgtm